network-api/networkapi/wagtailpages/static/js/directory-listing-filters.js

```diff
@@ -105,11 +105,16 @@
     });
 
     // And then we update the content that the user sees:
+    profileContainer.style.height = null;
     profileContainer.innerHTML = cards.join("\n");
     document.dispatchEvent(new CustomEvent("profiles:list-updated"));
   }
 
   function showLoadSpinner() {
+    let containerHeight = profileContainer.offsetHeight;
+    let containerFullHeight = `${containerHeight}px`;
+
+    profileContainer.style.height = containerFullHeight;
```

Comment:
Looking good!

Let's merge the three lines into 

```jsx
profileContainer.style.height = `${profileContainer.offsetHeight}px`
```

as the two variables `containerHeight` and `containerFullHeight` aren't used elsewhere & `profileContainer.offsetHeight` is pretty self explanatory.

Reply:
sounds good :)